src/main/java/com/rakovets/course/java/core/practice/oop/classes_and_objects/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package com.rakovets.course.java.core.practice.oop.classes_and_objects;
+
+public class Rectangle {
+   private Point firstPoint;
+   private Point secondPoint;
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"